### ISSUE TITLE
Want any help?

### ISSUE THREAD

**Issue by siebe41** [mikemountain/nfl-led-scoreboard#1]
Hey, found your project from the NHL and MLB ones. Was searching for something to base my own project off of to have something for a super bowl party. 

You seem fairly early on, would you like a hand with the development? Figured it would make sense to coordinate if you do.

**Comment by mikemountain** [comment]
Yeah man, that would be great. I've got a roadmap sort of lined out right
now but I think with so few games left in the season, it'd be best to just
kind of solidify the code that's there and clean it up a bit.

**Comment by siebe41** [comment]
Ok cool.
Let me know what you'd like me to tackle.
